shell/assets/translations/en-us.yaml

```diff
@@ -1731,8 +1732,9 @@ cluster:
       header: Security
     defaultPodSecurityPolicyTemplateName:
       label: Default Pod Security Policy
-    defaultPodSecurityAdmissionTemplateName:
+    defaultPodSecurityAdmissionConfigurationTemplateName:
       label: Default Pod Security Admission
+      option: RKE2 Default
```

Comment:
This doesn't appear to be used

Reply:
Good point. Let's wait for @kwwii feedback about this.

Reply:
I restored the default label for PSA and PSP.

Reply:
Small update: we finally come out with a decision and went for the option labels `RKE Default` as it was for PSP and `(None)` for PSA.